src/routes/safe/container/hooks/useFetchTokens.tsx

```diff
@@ -13,8 +13,9 @@ import { history } from 'src/store'
 export const useFetchTokens = () => {
   const dispatch = useDispatch()
   const address = useSelector(safeParamAddressFromStateSelector)
+  const { pathname } = history.location
```

Comment:
https://reacttraining.com/react-router/web/api/Hooks/uselocation you can use this instead of directly importing location
also please add types (void)

Reply:
It's done